river/ensemble/adaptive_random_forest.py

```diff
@@ -489,23 +455,21 @@ def __init__(self,
                  drift_detector: typing.Union[base.DriftDetector, None] = ADWIN(delta=0.001),
                  warning_detector: typing.Union[base.DriftDetector, None] = ADWIN(delta=0.01),
                  # Tree parameters
-                 max_size: int = 32,
-                 memory_estimate_period: int = 2000000,
                  grace_period: int = 50,
+                 max_depth: int = None,
                  split_criterion: str = 'info_gain',
                  split_confidence: float = 0.01,
                  tie_threshold: float = 0.05,
-                 binary_split=False,
-                 stop_mem_management=False,
-                 remove_poor_attrs=False,
-                 merit_preprune=True,
                  leaf_prediction: str = 'nba',
                  nb_threshold: int = 0,
                  nominal_attributes: list = None,
                  attr_obs: str = 'gaussian',
                  attr_obs_params: dict = None,
-                 max_depth: int = None,
-                 seed=None):
+                 max_size: int = 32,
+                 memory_estimate_period: int = 2000000,
+                 seed=None,
```

Comment:
Should be `seed: int = None`. It's not the end of the world not to specify it but it will help when we enforce `mypy` :)

Reply:
Done cb3e7f2835520439edcd5d1b3f3be9bee18cf278 :)